gcloud-java-storage/src/main/java/com/google/gcloud/storage/StorageImpl.java

```diff
@@ -577,6 +577,68 @@ public URL signUrl(BlobInfo blobInfo, long expiration, SignUrlOption... options)
     }
   }
 
+  @Override
+  public List<BlobInfo> get(BlobInfo blobInfo1, BlobInfo blobInfo2, BlobInfo... blobInfos) {
```

Comment:
Did you consider adding `method(BlobInfo)` to remove ambiguity and supporting `method(BlobInfo...)` to allow easy iteration conversion (similar to what you did for `Blob`)?